Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.cs

```diff
@@ -650,6 +709,90 @@ private static void OnFaceChanged(DependencyObject d)
             OnValueChanged(radialGauge);
         }
 
+        private void OnColorsChanged()
```

Comment:
Could this be done in the template with Theme Dictionaries?

Reply:
I'll create a specific test app for the narrator (it works nicely in the Toolkit Sample app, but that may indeed not be a representative host).

Can you rephrase the question on Theme Dictionaries, please?

Reply:
I have a new repo here: https://github.com/XamlBrewer/UWP-Control-Accessibility-Sample. If you start the app, activate Narrator, and use the TAB-key to navigate to the gauges, all of these are described. Maybe you're testing another scenario? If yes: which one?

Reply:
Just tested your sample and the sample app and it works as expected. (I must have had scan off on narrator initially, I think that's why it wasn't working).

On the Theme Dictionaries, is it possible to do [something like this](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.xaml#L5) in the template instead of the code behind?

Reply:
Working with Theme Dictionary now. Much better indeed, but it looks like I still need some code behind to let HighContrast override direct assignments like _controls:RadialGauge NeedleBrush="Red"_ in the client code.